presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/AgentFileUtils.java

```diff
@@ -71,10 +71,8 @@ public static void replaceFile(Path path, String url)
             throws IOException
```

Comment:
I would rename this method to `downloadFile`, and make its second argument a `URI`; let the caller create it however they like, not just as a string.